app/controllers/mixins/ems_common_angular.rb

```diff
@@ -267,7 +267,7 @@ def ems_form_fields
 
       if @ems.has_authentication_type?(:ssh_keypair)
         ssh_keypair_userid = @ems.authentication_userid(:ssh_keypair).to_s
-        ssh_keypair_auth_status = @ems.authentication_status_ok?(:ssh_keypair)
+        ssh_keypair_auth_status = @ems.kind_of?(ManageIQ::Providers::Redhat::InfraManager) ? true : @ems.authentication_status_ok?(:ssh_keypair)
```

Comment:
Couldn't this have been `@ems.authentication_status_ok?(:ssh_keypair)`?

The method is there, why should we ignore it for this specific provider?